### PR TITLE
Add support for gcloud_gcp_compute_network module

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject
Need to be merged after #1000 
As a dependency to gcloud_gcp_compute_router module

### CODE REVIEW

**Comment by samvarankashyap** [comment]
[test]

File: docs/source/examples/workspace/topologies/gcdns-record.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (14rcole) [inline docs/source/examples/workspace/topologies/gcdns-record.yml]: Same comment here as in #984

File: docs/source/examples/workspace/topologies/gce-net.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (14rcole) [inline docs/source/examples/workspace/topologies/gce-net.yml]: Did you forget to delete the file?

**Comment by samvarankashyap** [comment]
test succeeded but it shows failure. It might because of the consecutive force pushes.

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**